src/test/scala/io/citrine/lolo/trees/regression/RegressionTreeTest.scala

```diff
@@ -173,6 +175,24 @@ class RegressionTreeTest {
     /* They should have roughly the same value (linear regression has some noise in it) */
     assert(Math.abs(importances(0) - 0.30030792928576033) < 1.0e-3)
   }
+
+  /**
+    * Test with random weights
```

Comment:
"Test with random weights to ensure all feature importances of stump tree with linear leaves are non-negative."